app/src/common/key_and_signatures.c

```diff
@@ -141,27 +141,28 @@ static void format_signature_out(const uint8_t *signature)
     memmove(G_io_apdu_buffer + offset + 32 - xlength, signature + xoffset, xlength);
 }
 
-static int ecdsa_sign_hash_and_zero_out_private_key(
+static int ecdsa_sign_hash(
     cx_ecfp_private_key_t *privateKey,  // might be NULL if you do 'verify'
     cx_ecfp_public_key_t
         *public_key,  // might be NULL if you do 'sign' instead of 'verify'
     const unsigned char *in,
     unsigned short inlen, volatile unsigned char *out, unsigned short outlen,
-    unsigned char use_rfc6979_deterministic_signing) {
+    unsigned char use_rfc6979_deterministic_signing
```

Comment:
Would this be better as a `bool`, rather than `unsigned char`?  `bool` has been in C since at least C99.